rebasehelper/tests/testing_files/test.spec

```diff
@@ -15,6 +15,7 @@ Source2: ftp://test.com/test-source.sh
 Source4: file.txt.bz2
 Source5: documentation.tar.xz
 Source6: misc.zip
+Source7: https://pypi.python.org/packages/source/p/positional/positional-1.1.0.tar.gz
```

Comment:
This should refer to a local file, at least that's what `TestSpecFile.test_get_sources` method expects.